runtime-light/stdlib/web-transfer-lib/defs.h

```diff
@@ -64,64 +81,24 @@ class property_value {
-            if (v.is_vector()) {
-              tl::Vector<tl::WebPropertyValue> res{tl::vector<tl::WebPropertyValue>{.value = {}}};
-              for (const auto& i : v) {
-                if constexpr (std::is_same_v<value_t, array<bool>>) {
-                  res.inner.value.emplace_back(property_value::as_boolean(i.get_value()).serialize());
-                } else if constexpr (std::is_same_v<value_t, array<int64_t>>) {
-                  res.inner.value.emplace_back(property_value::as_long(i.get_value()).serialize());
-                } else if constexpr (std::is_same_v<value_t, array<string>>) {
-                  res.inner.value.emplace_back(property_value::as_string(i.get_value()).serialize());
-                }
-              }
-              return tl::WebPropertyValue{.value = std::move(res)};
-            } else {
-              tl::Dictionary<tl::WebPropertyValue> res{
-                  tl::Dictionary<tl::WebPropertyValue>{tl::vector<tl::dictionaryField<tl::WebPropertyValue>>{.value = {}}}};
-              for (const auto& i : v) {
-                // We cannot convert key into string right here since string which is produced will be destroyed once we occur out of this scope
-                kphp::log::assertion(i.get_key().is_string());
-                const auto& key{i.get_key().as_string()};
-                const auto& val{i.get_value()};
-                if constexpr (std::is_same_v<value_t, array<bool>>) {
-                  res.inner.value.value.emplace_back(
-                      tl::dictionaryField{.key = tl::string{{key.c_str(), key.size()}}, .value = property_value::as_boolean(val).serialize()});
-                } else if constexpr (std::is_same_v<value_t, array<int64_t>>) {
-                  res.inner.value.value.emplace_back(
-                      tl::dictionaryField{.key = tl::string{{key.c_str(), key.size()}}, .value = property_value::as_long(val).serialize()});
-                } else if constexpr (std::is_same_v<value_t, array<string>>) {
-                  res.inner.value.value.emplace_back(
-                      tl::dictionaryField{.key = tl::string{{key.c_str(), key.size()}}, .value = property_value::as_string(val).serialize()});
-                }
-              }
-              return tl::WebPropertyValue{.value = std::move(res)};
-            }
-          } else {
-            static_assert(false);
-          }
-        },
-        this->value);
+  static inline auto as_array_of_string(array<string> value) noexcept -> property_value {
+    return property_value{std::move(value)};
   }
+
+  inline auto serialize() const noexcept -> tl::webPropertyValue;
+  static inline auto deserialize(const tl::webPropertyValue& tl_prop_value) noexcept -> property_value;
+
+  inline auto to_mixed() const noexcept -> mixed;
 };
 
+using properties_type = kphp::stl::unordered_map<property_id, property_value, kphp::memory::script_allocator>;
```

Comment:
It has single usage in `simple_transfer_config`. Why isn't this declared within it?